Realm/Realm/Realm.cs

```diff
@@ -171,30 +171,23 @@ public static bool Compact(RealmConfigurationBase config = null)
         /// <summary>
         /// Deletes all the files associated with a realm.
         /// </summary>
+        /// <remarks>
+        /// The realm file must not be open on other threads.
+        /// This method must not be called inside a transaction.
+        /// All but the .lock file will be deleted by this.
+        /// </remarks>
         /// <param name="configuration">A <see cref="RealmConfigurationBase"/> which supplies the realm path.</param>
+        /// <exception cref="RealmPermissionDeniedException">Thrown if the Realm is still open.</exception>
         public static void DeleteRealm(RealmConfigurationBase configuration)
         {
             Argument.NotNull(configuration, nameof(configuration));
 
-            var fullpath = configuration.DatabasePath;
-            if (IsRealmOpen(fullpath))
+            if (IsRealmOpen(configuration.DatabasePath))
```

Comment:
You can remove this call now that Core takes care of performing the check.

Reply:
I wasn't sure if we wanted to keep throwing our custom exception here.
Fixed with 84be8d0a1769fed6159dd3cbdcba0ae0dc833d9f.
The test checks for a generic `RealmException` now. Would we want to test the specific exception thrown by core? I guess not.